File: aulas/aula_05/listapoo01_q3.java
class Conta {
    public String titular;
    public String numero;
    public double saldo;

    public Conta(String titular, String numero, double saldo) {
        this.titular = titular;
        this.numero = numero;
        this.saldo = saldo;
    }

    public void depositar(double saldo) {
        this.saldo += saldo;
    }

    public void sacar(double saldo) {
        this.saldo -= saldo;
    }
}

public class listapoo01_q3 {
    public static void main(String[] args) {
        Conta conta1 = new Conta("Wenderson", "123-X", 2000);
        System.out.println(conta1.titular + " " + conta1.numero);
    }
}

File: aulas/aula_05/listapoo01_q3.cpp
#include <iostream>
using namespace std;

class Conta {
    public:
    string titular;
    string numero;
    double saldo;

    Conta(string titular, string numero, double saldo) {
        this->titular = titular;
        this->numero = numero;
        this->saldo = saldo;
    }

    void depositar(double valor) {
        saldo += valor;
    }

    void sacar(double valor) {
        saldo -= valor;
    }
    
};

int main() {
    Conta x("Wenderson", "123-X", 2938); // É uma instância | A instância possui o construtor da 
    // Conta& y = x; // y É uma referência para x
    // Conta* z = new Conta("Wenderson"); // z é um ponteiro para uma instância


    cout << "Titular: " << x.titular << endl;
    cout << "Numero: " << x.numero << endl;
    cout << "Saldo: " << x.saldo << endl;
    return 0;
}